RecoEgamma/EgammaPhotonProducers/src/GEDPhotonProducer.cc

```diff
@@ -183,6 +185,8 @@ class GEDPhotonProducer : public edm::stream::EDProducer<edm::GlobalCache<CacheD
 
   //MIP
   std::unique_ptr<PhotonMIPHaloTagger> photonMIPHaloTagger_ = nullptr;
+  //MVA based Halo tagger for the EE photons
+  std::unique_ptr<PhotonMVABasedHaloTagger> photonMVABasedHaloTagger_ = nullptr;
```

Comment:
it looks to me that per this implementation, the PhotonMVABasedHaloTagger is not in the cache, thus the GBR memory is duplicated for each thread. Is there any reason not to put it in the cache instead?